database/src/main/kotlin/no/nav/su/se/bakover/database/revurdering/RevurderingPostgresRepo.kt

```diff
@@ -481,6 +487,7 @@ internal class RevurderingPostgresRepo(
                         is RevurderingTilAttestering.Innvilget -> RevurderingsType.TIL_ATTESTERING_INNVILGET
                         is RevurderingTilAttestering.Opphørt -> RevurderingsType.TIL_ATTESTERING_OPPHØRT
                     },
+                    "sendBrev" to if (revurdering is RevurderingTilAttestering.IngenEndring) revurdering.sendBrev else null
```

Comment:
Er det noen grunn til at dette ikke bare settes til `true` i de tilfellene man ikke skal ha mulighet til å velge det bort? Jeg tipper at det er fordi det ikke er relevant i andre caser enn `IngenEndring`, men jeg tenker at det kanskje ser litt forvirrende ut at man ser `null`-verdier i databasen.

Reply:
Føles vel litt iffy om `sendBrev = null` i realiteten betyr det samme som `sendBrev = true`. Regner med det er for å begrense omfanget til ingen endring?

Reply:
Jeg tenkte feltet kunne representere et manuelt valg, som i utgangspunktet kun gjelder for "ingen endring" (siden det er en potet-tabell). Siden de fleste tilfeller til nå ikke trenger å bry seg om dette feltet for å vite om det skal sende brev eller ikke. null representerer de tilfellene det ikke har blitt tatt et manuelt valg. Her kunne man så klart brukt en sealed class og persistert det som json for å være tydeligere. Dersom det er true eller false, vil vi da vite at saksbehandler har tatt stilling til det.

En ting vi enda ikke har løst er historiske revurderinger som har ført til IngenEndring. Disse forventer vi at er true eller false. Foreslår at vi migrerer de til false. Hva synes dere?

Reply:
Revurderinger med status IngenEndring har bare vært i test siden i går, et alternativ er også å slette de enten via sql-migreringa eller manuelt.

Reply:
Enig i at det høres fornuftig ut å migrere de eksisterende `IngenEndring` til `false` ja.
Ser poenget med at `null` betyr at det ikke har blitt tatt et valg, men det kan også føles som at "her mangler vi fortsatt litt informasjon fra saksbehandler". Jeg tror jeg fortsatt heller mot at det ikke kan være nullable, og at i de tilfellene der det alltid skal sendes brev så hardkodes det bare til `false`.

Reply:
@ingara Du mente kanskje at `sendBrev` skulle hardkodes til `true` hvis det skal sendes brev?